README.md

```diff
@@ -1,36 +1,91 @@
+                </br>Safari
+            </td>
+            <td align="center">
+                <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/opera/opera_48x48.png" alt="Opera" width="24px" height="24px"/>
+                </br>Opera
+            </td>
+            <td align="center">
+                <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/brave/brave_48x48.png" alt="Brave" width="24px" height="24px"/>
+                </br>Brave
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Downloads
+[![GitHub](https://img.shields.io/badge/-github-222222?style=for-the-badge&logo=github)](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/raw/main/dist/Simple-YouTube-Age-Restriction-Bypass.user.js)
+[![GreasyFork](https://img.shields.io/badge/-greasyfork-950000?style=for-the-badge&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAQAAAC1+jfqAAAAAXNSR0IArs4c6QAAAYdJREFUeNpNkU1LVHEUh3+jn8GdtYg+QOHiCiESBokQFaFQYBFYGAlGZMnQVLNIVFB8uQPqoCPeq+JCdBgUHXUUBF9WioLrOzsXLl0M18v/Uf6oeB44HM554Cx+klQmSaoe8SqD4bA7/BH0+ZWO7OWmxb6kRxmkjjlyLLDFBPdcYjeK2rbesm46IpkanvDRvIm+GgcVrCK1pj+RLzWQp516niJLTelqcu3vIZbNB7L85JQZFkizQj8tzBufhCN538hEi/zlHDhgGP+K/7QxFQ3Q4UmBkElyBsAR4jHCYoQCNYbveE4VGY45AfI84wV1vEb0kAw1frHLfXrpp4teq2wjchSoZ4LJUPFiHJn3tJLE5RX7wBrlNCLzm85AZZ5QJGrRNXvAKnaLPDU7CVKmk2myeGywzi+rrPDZJJAjyXU9ZktLZJkhS4414hxCaZOBlGzFqgq1NJlHUYV5STMZ0xTJ5JnfuROWXFHBA2RpoQGlVH59/We9h9Xf/a5iOpwM/xTH/PbbuC8B++4oDDg4lrsAAAAASUVORK5CYII=)](https://greasyfork.org/en/scripts/423851-simple-youtube-age-restriction-bypass)
+[![OpenUserJS](https://img.shields.io/badge/-openuserjs-2C3E50?style=for-the-badge)](https://openuserjs.org/scripts/zerodytrash/Simple_YouTube_Age_Restriction_Bypass)
+
+</div>
 
-3. Add "Simple-YouTube-Age-Restriction-Bypass.user.js" by clicking [this link](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/raw/main/dist/Simple-YouTube-Age-Restriction-Bypass.user.js).
-4. Done.
+## How it works
 
+When the video is detected as restricted, the bypass will retrieve the video and all other information from YouTube's unrestricted ``/youtubei/v1/player`` endpoint. As a fallback (for some non-embeddable videos) an open source <a href="https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/tree/main/account-proxy">Account Proxy Server</a> is used. This allows you to watch age-restricted videos without verification and the need to be signed in.
+
+All videos will be unlocked automatically.
+
+## Installation
+1. Install a userscript manager of your choice:<br>
+&bull; [Tampermonkey](https://www.tampermonkey.net)<br>
+&bull; [Violentmonkey](https://violentmonkey.github.io/get-it)<br>
+&bull; [Greasemonkey](https://addons.mozilla.org/en-US/firefox/addon/greasemonkey)<br>
+
+2. Install Simple YouTube Age Restriction Bypass by clicking on one of the [downloads](#downloads).
```

Comment:
Should we rather link directly to the userscript from the dist folder? I think that would be easier. Some users may not understand the anchor link.

Reply:
I thought the downloads section was clear enough that people would know. 
But feel free to change it as you see fit! ;)